modules/installation-configuration-parameters.adoc

```diff
@@ -485,6 +485,17 @@ ifdef::rhv[]
 For details, see the "Additional RHV parameters for machine pools" table.
 endif::rhv[]
 
+ifdef::openshift-origin[]
```

Comment:
Just a small structure nit, this `ifdef` should go after the `endif::openshift-origin[]` on line 528. So all the `ifdefs` with similar content will be together